crates/ruff_python_formatter/tests/snapshots/black_compatibility@miscellaneous__long_strings_flag_disabled.py.snap

```diff
@@ -316,37 +316,23 @@ long_unmergable_string_with_pragma = (
  
  comment_string = "Long lines with inline comments should have their comments appended to the reformatted string's enclosing right parentheses."  # This comment gets thrown to the top.
  
-@@ -165,25 +165,13 @@
+@@ -165,13 +165,9 @@
  
  triple_quote_string = """This is a really really really long triple quote string assignment and it should not be touched."""
  
 -assert (
 -    some_type_of_boolean_expression
 -), "Followed by a really really really long string that is used to provide context to the AssertionError exception."
-+NOT_YET_IMPLEMENTED_StmtAssert
++assert some_type_of_boolean_expression, "Followed by a really really really long string that is used to provide context to the AssertionError exception."
```

Comment:
Ehh, this is interesting. It seems black happily wraps identifiers in parentheses in *some* positions but not all. This is fine for now.

Reply:
i wonder if that's because assert is somewhat unusual in that one must not wrap the entire rhs in parens, i.e. 

`assert foo, "msg"` very much != `assert (foo, "msg")`